nixos/modules/misc/ids.nix

```diff
@@ -355,7 +355,6 @@ in
       rstudio-server = 324;
       localtimed = 325;
       automatic-timezoned = 326;
-      whisparr = 328;
```

Comment:
Why this change and the one below? It seems unrelated to this PR.

Reply:
Oops, looks like I bungled that when migrating to `DynamicUser` 👀 
Thanks for noticing, should be fixed now.